src/psyclone/tests/psyir/transformations/intrinsics/abs2code_trans_test.py

```diff
@@ -73,13 +73,10 @@ def example_psyir(create_expression):
 
     '''
     symbol_table = SymbolTable()
-    name1 = symbol_table.new_symbol_name("arg")
-    arg1 = DataSymbol(name1, REAL_TYPE, interface=ArgumentInterface(
-        ArgumentInterface.Access.READWRITE))
-    symbol_table.add(arg1)
-    name2 = symbol_table.new_symbol_name()
-    local = DataSymbol(name2, REAL_TYPE)
-    symbol_table.add(local)
+    arg1 = symbol_table.new_symbol(
```

Comment:
Please could you update the copyright date on this file.

Reply:
Done